tools/custom-checks/run-attw-on-tgz.ts

```diff
@@ -0,0 +1,44 @@
+import esMain from "es-main";
+import { getAllTgzPath } from "./get-all-tgz-path";
+import { exec } from "child_process";
+import { exit } from "process";
+
+let attwFailedPackages: string[] = [];
+
+export async function runAttwOnTgz(): Promise<void> {
+  const TAG = "[tools/custom-checks/run-attw-on-tgz.ts]";
+  console.log(`${TAG} Fetching .tgz file paths.`);
+
+  {
+    const { relativePaths: tgzFilesRelative } = await getAllTgzPath();
+    console.log(`${TAG} Found ${tgzFilesRelative.length} .tgz files.`);
+    for (const filePath of tgzFilesRelative) {
+      await execCommand("attw", filePath);
+    }
+  }
+
+  function execCommand(binaryName: string, filePath: string): Promise<void> {
```

Comment:
You can use `promisify` from `node:util` to simplify this function (by awaiting the exec method). Or, you could just use execSync since we await for each file anyway